<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RTL SDR Spectrum Analyzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="6,0">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item row="2" column="1">
     <widget class="QLabel" name="lblFreq">
      <property name="text">
       <string>100.0Mhz</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>1</number>
      </property>
      <widget class="PlotWidget" name="spectrumView"/>
      <widget class="PlotWidget" name="waterfallView"/>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QToolBox" name="toolBox">
      <property name="minimumSize">
       <size>
        <width>255</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>255</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>255</width>
         <height>470</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Tuner</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>231</width>
          <height>121</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Gain (dB)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBoxGain">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentText">
            <string>38.6</string>
           </property>
           <property name="currentIndex">
            <number>21</number>
           </property>
           <item>
            <property name="text">
             <string>0.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>0.9</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1.4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2.7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3.7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7,7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8.7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12.5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>14.4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15.7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16.6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19.7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>20.7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>22.9</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>25.4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>28.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>29.7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>32.8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>33.8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>36.4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>37.2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>38.6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>40.2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>42.1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>43.4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>43.9</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>44.5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>48.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>49.6</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>NFFT</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBoxNFFT">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentText">
            <string>1024</string>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>1024</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2048</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4096</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8192</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="spinBoxPPM">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>200</number>
           </property>
           <property name="value">
            <number>60</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Freq Correction</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="pushButtonConnect">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>133</y>
          <width>231</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>234</width>
          <height>164</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Center Frequency (Hz)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBoxFrequency">
           <property name="minimum">
            <number>20000000</number>
           </property>
           <property name="maximum">
            <number>2000000000</number>
           </property>
           <property name="singleStep">
            <number>100000</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Averaging</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBoxAlfa">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.001000000000000</double>
           </property>
           <property name="stepType">
            <enum>QAbstractSpinBox::DefaultStepType</enum>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>FFT Window</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBoxWindow">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentText">
            <string>Hanning</string>
           </property>
           <property name="currentIndex">
            <number>4</number>
           </property>
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bartlett</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Blackman</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hamming</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hanning</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSpinBox" name="spinBoxGenFrequency">
           <property name="minimum">
            <number>100000</number>
           </property>
           <property name="maximum">
            <number>30000000</number>
           </property>
           <property name="singleStep">
            <number>1000</number>
           </property>
           <property name="value">
            <number>20000000</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>DDS Frequency (Hz)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="radioButtonADC">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>ADC in</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QRadioButton" name="radioButtonGenerator">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>DDS Generator</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="toolBoxLevel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>255</width>
         <height>470</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Waterfall</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
